backend/src/auth/utils.py

```diff
@@ -117,11 +126,16 @@ def encode_jwt_token(username: str, user_id: uuid.UUID) -> dict:
     return data
 
 
-def get_user_id_from_request(request: Request) -> Optional[uuid.UUID]:
+def get_token_from_request(request: Request) -> Optional[str]:
     authorization = request.headers.get("Authorization")
-    if not authorization:
-        return None
-    auth_token = authorization.split(" ")[1]
+    scheme, token = get_authorization_scheme_param(authorization)
```

Comment:
Authorization or authentication ?

Reply:
Authorization is ok. get_authorization_scheme_param is from fastapi security.

Reply:
raise is wrong shood be AuthorizationFailed